images/base/Makefile

```diff
@@ -0,0 +1,27 @@
+TAG?=$(shell echo "$$(date +v%Y%m%d)-$$(git describe --always --dirty)")
+IMAGE?=kindest/base:$(TAG)
+
+# build with buildx
+PLATFORMS?=linux/amd64,linux/arm64
```

Comment:
we'll probably want to expand this, however I'm trying to wrap this up and make sure the base image works at all, my main priority was the containerd update, this change is more of a diversion due to #1347 #166